src/Network/Network/help/Remove-AzExpressRoutePortIdentity.md

```diff
@@ -0,0 +1,112 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/az.network/remove-azexpressrouteportidentity
+schema: 2.0.0
+---
+
+# Remove-AzExpressRoutePortIdentity
+
+## SYNOPSIS
+Removes a identity from an ExpressRoutePort.
+
+## SYNTAX
+
+```
+Remove-AzExpressRoutePortIdentity -ExpressRoutePort <PSExpressRoutePort>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
```

Comment:
Cmdlet doesn't support `ShouldProcess`, please remove `WhatIf` and `Confirm` from readme.

Reply:
Done